Documentation/commands.md

```diff
@@ -81,8 +87,8 @@ or `rkt list --full`
 
 ```
 # rkt list --full
-UUID					                APP	    ACI 	STATE	NETWORKS
-f241c969-1710-445a-8129-d3a7ffdd9a60	busybox	busybox	running
+UUID					APP	IMAGE NAME		IMAGE ID		STATE	NETWORKS
+132f9d56-0e3f-4d1e-ba86-68efd488bb62	etcd	coreos.com/etcd:v2.0.10 sha512-c03b055d02e5	running
 ```
 
 Pod's machine name will be the pod's UUID with a `rkt-` prefix.
```

Comment:
The pod's machine name will be the pod's UUID prefixed with `rkt-`. Given this machine name, logs can be retrieved by `journalctl`: